### DISCUSSION TITLE
How use Matomo Analytics on my own Datasette instance

### DISCUSSION THREAD

**Discussion by nicfab** [simonw/datasette#2428] [Q&A]
I set up a Datasette instance on my server (Ubuntu 20.04).
I want to add a Matomo script for the analytics.
How can I do it?

**Comment by clombion** [comment]
There is a plugin for that https://github.com/CentreForDigitalHumanities/datasette-matomo

**Reply by nicfab** [reply]
> There is a plugin for that https://github.com/CentreForDigitalHumanities/datasette-matomo

Thank you. 
I see that I can set Matonomo in the environment (https://github.com/CentreForDigitalHumanities/datasette-matomo?tab=readme-ov-file#usage).
Should I put the environment values in the `settings.json` file?
